rd-net/RdFramework/Base/RdExtBase.cs

```diff
@@ -202,7 +204,7 @@ public void Send<TContext>(RdId id, TContext param, Action<TContext, UnsafeWrite
     {
       lock (mySendQ)
       {
-        if (mySendQ.Count > 0 || !Connected.Value)
+        if (!RealWire.IsStub && (mySendQ.Count > 0 || !Connected.Value))
```

Comment:
For performance reasons, you may want to extract this new check outside of `lock`, to completely avoid locking on `mySendQ` for local connections.

Reply:
yep, it also does not make sense to call send of the real wire when it is stub. Fixed, thanks